server/src/common/dto/query/map-queries.dto.ts

```diff
@@ -176,3 +176,35 @@ export class MapRanksGetQuery extends PaginationQuery {
     @IsOptional()
     orderByDate: boolean;
 }
+
+export class MapRankGetNumberQuery {
+    @ApiPropertyOptional({
+        name: 'trackNum',
+        type: Number,
+        description: 'Track number (default 0)'
```

Comment:
You can use the `default` property here so `default: 0` which swagger will show, better than typing the default manually

Reply:
Applies in a couple cases below